website/docusaurus.config.ts

```diff
@@ -108,6 +108,11 @@ const config: Config = {
           label: "Typedoc",
```

Comment:
General thought (not something we need to update now) - should this just be "TypeScript Docs"? Is TypeDoc a well known thing? Saying TS might be clearer.

Might be more in line with the "Solidity Docs" label (e.g. we're not calling it "Docgen")

Reply:
sure, I like that idea